packages/devextreme/js/localization/messages/cs.json

```diff
@@ -12,6 +12,8 @@
         "Search": "Hledat",
         "Back": "Zpět",
         "OK": "OK",
+        "Today": "Today",
```

Comment:
We already have the transcription for the word 'today' in all languages. See 'dxCalendar-todayButtonText.' Why don't we use this knowledge to localize this variable immediately?

@ksercs What do you think, should we do this right away?

Reply:
Yeah, it's better to use translations if we already have it

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925